### PR TITLE
Fix school page

### PR DESCRIPTION
**Pull request by jleong01**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html [hunk 1]
```diff
@@ -1 +1,3 @@
 <div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
+<div data-sly-test.referencePage="${pageProperties.referencePage}"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: I am still un-sure about the requirement of this change. After this, there will be two parsys inserted.

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: as per TD

If this path is entered, the parsys of the current page will be using a sling:include of the parsys of the target page.
E.g. if the path is “/content/xx/yy/zz”, the parsys will be using the sling:include path of “/content/xx/yy/zz/jcr:content/par”

File: ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html [hunk 2]
```diff
@@ -1 +1,4 @@
-<div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
\ No newline at end of file
+<div data-sly-resource="${'par' @resourceType='wcm/foundation/components/responsivegrid'}"></div>
+<div data-sly-test.referencePage="${pageProperties.referencePage}"
```
Comment (bsinghal1) [inline ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: don't put this change to the basepage/content.html. Put this into the schoolpage/content.html

Comment (jleong01) [reply ui.apps/src/main/content/jcr_root/apps/taylorsuniversity/components/page/basepage/content.html]: moved code snippet to schoolpage/content.html

**Pull request merged**